qiskit_ibm_runtime/ibm_backend.py

```diff
@@ -492,10 +509,25 @@ def __call__(self) -> "IBMBackend":
         # For backward compatibility only, can be removed later.
         return self
 
-    def run(self, *args: Any, **kwargs: Any) -> None:
-        """Not supported method"""
-        # pylint: disable=arguments-differ
-        raise RuntimeError("IBMBackend.run() is not supported in the Qiskit Runtime environment.")
+    def _check_circuits_attributes(self, circuits: List[QuantumCircuit]) -> None:
+        """Check that circuits can be executed on backend.
+        Raises:
+            IBMBackendValueError:
+                - If one of the circuits contains more qubits than on the backend."""
+
+        if len(circuits) > self._max_circuits:
+            raise IBMBackendValueError(
+                f"Number of circuits, {len(circuits)} exceeds the "
+                f"maximum for this backend, {self._max_circuits})"
+            )
+        for circ in circuits:
+            if isinstance(circ, QuantumCircuit):
```

Comment:
Type hint needs to be fixed if the input isn't always `QuantumCircuit`

Reply:
Actually, this check is unnecessary, so I removed the `isinstance` line.

Reply:
On further checking, there is one test `test_single_openqasm3_submission` that submits a string, so I put back the check and updated the hint.